implementation/src/main/java/io/smallrye/metrics/legacyapi/LegacyMetricsExtension.java

```diff
@@ -6,25 +6,10 @@
 import java.lang.reflect.Modifier;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
+import java.util.*;
```

Comment:
should we keep it explicit and not use a wildcard?

Reply:
Agreed - I've pushed a change for this.